includes/variables.php

```diff
@@ -105,6 +105,17 @@
 //load the geocoding method
 $tsml_geocoding_method = get_option('tsml_geocoding_method', 'legacy');
 
+//options for geocoding
+$tsml_geocoding_methods = [
+	'legacy' => __('Legacy Method', '12-step-meeting-list'),
+	'api_gateway' =>  __('API Gateway (Alpha, temporary)', '12-step-meeting-list'),
+	'c4r_geocoding' =>  __('API Gateway (Beta)', '12-step-meeting-list'),
```

Comment:
I suggest removing `api_gateway` as an option the user can select. Today we only want them testing the new service. We could even automatically shift those users with the alpha test site selected to the new beta?

Second suggestion: Change 112 to `'c4r_geocoding' => __('C4R Geocoding (Beta)', '12-step-meeting-list'),`